ROADMAP.md

```diff
@@ -0,0 +1,87 @@
+But it is not obvious how to obtain relevant metrics from that data,
+in a way that is useful and actionable.
+
+## Mission (overarching goal):
+
+GMD work group is developing metrics, methodologies, and software
+for expressing open source project health and sustainability.
+By measuring open source project health and sustainability,
+GMD (Growth-Maturity-Decline) seeks to improve the transparency of open source project
+health and sustainability so that relevant stakeholders can
+make more informed decisions about open source project engagement.
+
+## Risks:
+
+* No one wants to use GMD metrics.
+* No other software implements GMD metrics.
+* Use cases do not resonate with people.
+
+## Goals
+
+### Goal #0: Make the boundaries of the GMD Focus area more clear and specific
+
+### Goal #1: Fully Implemented Focus Areas
+
+We have to produce a full version with all the metrics for at least some of the focus areas.
+
+* We are aiming to have at least one of the focus areas implemented from top to bottom
+by February, including reference implementations. Thus area will be code development.
+
+* We are aiming to produce metrics for the next CHAOSSCon (February), in at least 2 focus areas.
+
+* We are aiming to release formally version 1 of the metrics by Summmer.
+
+### Goal #2: Make focus areas consistent with use cases
+
+Use cases may not immediately apply to a specific area,
+so they can be specified without being in a focus area at first. 
+
+Focus areas already take into account some use cases,
+and this list of use cases that apply can be enumerated under
+the focus area (and cross referenced where applicable),
+and then elaborated as full use cases later. 
+
+* We are aiming to list use cases for each focus area.
+
+* We are aiming to identify candidate focus areas for existing use cases. 
+
+* We are aiming in general to make focus areas and use cases more consistent with each other.
+Through a process of zooming out (enumerated use cases)
+and zooming in (elaborating specific use cases),
+the focus areas will become more consistent with use cases and vice versa. 
+
+### Goal #3: System status document/dashboard
+
+We need a way to know which metrics are implemented by CHAOSS projects,
+a kind of a dashboard or tables in a document where people can quickly
+understand wich metrics are already implemented.
+
+* We are aiming at lists of concrete implementations of CHAOSS metrics
+in Augur and GrimoireLab.
+
+* We are aiming to have reference implementations for as much of the metrics
+as possible, in the implementation directory.
+
+### Goal #4: Have more use cases
+
+We need use cases to inform how people are using, or intending to use,
+metrics for understanding their projects.
+This should be one of the sources of information to write our
+goal-question-metric analysis in the focus areas documents.
+
+* We are aiming to have more use cases fully described.
+
+### Goal #5:  Grow the work group
```

Comment:
Goal #5: Maybe GMD could also do some introspection and document workflows to make them accessible to new members. Maybe that's part of the 'onboarding documents', but I think of it more as governance documents that guide every-day work.